test/io/ferguson/writef.chpl

```diff
@@ -1,3 +1,4 @@
+pragma "error mode fatal"
```

Comment:
I was just reviewing this to see what tests needed to be updated with this switch.  Is it the use of writef()s in this test that requires a switch or something else?

Reply:
Actually this one doesn't really need the switch (it did in an earlier version of the PR, when I had the top-level writef throw - it does not now).